examples/jsm/renderers/common/Background.js

```diff
@@ -59,8 +59,11 @@ class Background extends DataMap {
 					getSamplerLevelNode: () => backgroundBlurriness
 				} ).mul( backgroundIntensity );
 
-				let viewProj = modelViewProjection();
-				viewProj = viewProj.setZ( viewProj.w );
+				const viewProj = tslFn( () => {
```

Comment:
This wouldn't work normally, because `currentStack` is null when this is ran -- so I've wrapped it in a some kind of "Node IIFE".